app/models/labware_creators/pooled_tubes_from_whole_tubes.rb

```diff
@@ -11,7 +11,6 @@ class SubmissionFailure < StandardError; end
 
     self.page = 'pooled_tubes_from_whole_tubes'
     self.attributes += [{ barcodes: [] }]
-    self.default_transfer_template_name = 'Whole plate to tube'
 
```

Comment:
Not relevant for this creator

Reply:
Just noticed the comment at the top of this file says `Pools an entire plate into a single tube. Useful for MiSeqQC`
Presume should say `Pools an entire tube into a single tube. Useful for MiSeqQC`

Reply:
Actually that line should be completely re-written, its left over from another creator that I used to template this one.

Reply:
Updated the comment to remove references to plates (And MiSeq QC)